packages/i18n-calypso/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 7.0.0
+
+- Update React peer dependency to v18.
+- Improve TypeScript type definitions.
```

Comment:
May be nice to include a few more items that happened since the last release: https://github.com/Automattic/wp-calypso/commits/67d6f57001329952de28b4bae87f1689b899daf1/packages/i18n-calypso

Reply:
I went through the diff since the 6.0.1 release and added a few more dependency updates that seemed relevant. I don't see anything else that's worth mentioning in the change log.